src/web/core/src/main/java/org/geoserver/web/ServiceLinkDescription.java

```diff
@@ -83,6 +104,10 @@ public String getProtocol() {
         return protocol;
     }
 
+    public String getSpecificServiceType() {
```

Comment:
Tying to document the new term as it is not clear to me how this differs from protocol:
```suggestion
    /**
      * While serviceType used establish a group with title and description, the specific service type can identify
      * identify the service type used for configuration.
      */
    public String getSpecificServiceType() {
```
```suggestion
    public String getSpecificServiceType() {
```
```suggestion
    public String getSpecificServiceType() {
```
```suggestion
    public String getSpecificServiceType() {
```

Reply:
Protocol is what is actually put on the screen - i.e. OGCAPI-Features in my first picture, and just "Features" in my second.